src/index.css

```diff
@@ -1,27 +1,247 @@
+}
+
+.header a {
+  color: #121212;
+  text-decoration: none;
+}
+
+.header .userOval {
+  display: flex;
+  border: 1px solid #e8e8e8;
+  border-radius: 50%;
+  margin-left: auto;
+  padding: 10px;
+}
+
+.header .userAvatar {
+  padding: 15px;
+  height: 30px;
+}
+
+.addBookForm-container {
+  flex-direction: column;
+  gap: 1rem;
+  display: flex;
+  padding-bottom: 3rem;
+  margin: 3rem 6rem 0 6rem;
+}
+
+.addBookForm-container input {
+  width: 80%;
+}
+
+.addBookForm-container h2 {
+  border-top: 1px solid #e8e8e8;
+  color: rgb(87, 87, 87);
+  padding-top: 1em;
+  margin: 2em 0 1em;
+}
+
+.categoryInput {
+  border: solid 1px #e8e8e8;
+  padding: 10px;
+  margin-right: 20px;
+  background: white;
+  width: 50%;
+}
+
+.bookContainer {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 6px 6px 12px #beb8b8, -6px -6px 12px #fff;
+  background: #fff;
+  border-radius: 2px;
+  padding: 4rem 2.5rem;
+}
+
+.bookCategory {
+  padding: 0.5rem 0;
+  opacity: 0.9;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #e8e8e8;
+}
+
+.bookTitle {
+  padding: 0.5rem 0;
+  font-family: RobotoSlab;
+  font-size: 2.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  color: #121212;
+}
+
+.bookAuthor {
+  padding: 0.5rem 0;
+  font-family: RobotoSlab;
+  font-size: 1.475rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.bottomLinks {
+  margin-top: 0.5rem;
+  cursor: pointer;
+  color: #4386bf;
+  gap: 1rem;
+}
+
+.Line-2 {
+  height: 2.4rem;
+  margin: 0.75rem 1.063rem 0 0.938rem;
+  border: solid 1px #e8e8e8;
+}
+
+.Line-3 {
+  height: 60%;
+  margin: 0.75rem 1.063rem 0 0.938rem;
+  border: solid 1px #e8e8e8;
+}
+
+.bookRemoveButton {
+  border: none;
+  background: none;
+  color: #4386bf;
+  font-family: RobotoSlab;
+  font-size: 1.475rem;
+  font-weight: 400;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  cursor: pointer;
+}
+
+.completedness {
+  width: 50%;
+  display: flex;
+  padding: 0 2rem;
+  place-items: center;
+  flex-direction: row;
+  justify-content: space-around;
+  justify-items: center;
+  align-items: center;
+}
+
+.barinfo {
+  width: 2rem;
+  margin: 0;
+  line-height: normal;
+  font-family: Montserrat, sans-serif;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  padding: 0;
+  letter-spacing: normal;
+  color: black;
+}
+
+.bar {
+  width: 10rem;
+  display: flex;
+  flex-direction: row;
+}
+
+.progress {
+  gap: 6px;
+}
+
+.updateProgressButton {
+  font-weight: 600;
+  color: white;
+  background-color: #4395fc;
+  border: none;
+  border-radius: 5px;
+  padding: 10px 20px;
+  cursor: pointer;
+  box-shadow: 5px 5px 5px rgb(0, 0, 0, 0.3);
+}
+
+.bookList {
+  margin: 3rem 6rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
```

Comment:
- [ ] The booklist container has a `background-color` of `#fafafa` that is different from the `background-color` of each book. It also has a border and padding that makes it look like this:

### Zeplin

![image](https://user-images.githubusercontent.com/77106962/208261606-254b31f1-5471-40da-9db0-419f8e47d218.png)

#### This is the way yours look

### Your Design

![image](https://user-images.githubusercontent.com/77106962/208261992-5b11125c-e3c8-407a-b0e1-9be40d43fe7f.png)

Kindly implement this change. 👍